lib/ansible/modules/cloud/amazon/aws_codedeploy_application.py

```diff
@@ -83,6 +83,9 @@
     description:
       - Information about the load balancer used in a deployment.
     required: false
+extends_documentation_fragment:
+    - aws
+    - ec2
```

Comment:
Those ec2_* warnings are related to this documentation fragment.

Reply:
I added that after I saw the errors and did some research on it. They appeared even without it and I thought that that inclusion would silence those errors, unless I'm doing it wrong, I'm not really sure why they're still appearing. It is odd that ansibot said it failed when the Shippable test below shows it passed. Maybe I'm confused, but I don't know what else to do.

Reply:
As you say, shippable is showing all the tests as passing - ansibot must have still been processing the previous shippable failure.